nservicebus/sql-persistence/index.md

```diff
@@ -16,6 +16,8 @@ The SQL Persistence uses [Json.NET](http://www.newtonsoft.com/json) to serialize
  * [SQL Server](https://www.microsoft.com/en-au/sql-server/) ([Version 2012](https://docs.microsoft.com/en-us/sql/release-notes/sql-server-2012-release-notes) and above due to the use of the [THROW functionality](https://msdn.microsoft.com/en-us/library/ee677615.aspx)).
  * [MySQL](https://www.mysql.com/) ([Version 5.7](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/) and above due to the use of [JSON Data Type](https://dev.mysql.com/doc/refman/5.7/en/json.html)).
 
+WARNING: This persistence will run on the free version of the above engines, i.e. [SQL Server Express](https://www.microsoft.com/en-au/sql-server/sql-server-editions-express) and [MySQL Community Edition](https://www.mysql.com/products/community/). However, due to the benefits of support, it is strongly recommended to purchase a paid version for running any production system. Its is also recommended to ensure that support agreements are in place from either [MySQL support](https://www.mysql.com/support/), [Microsoft Premier Support](https://www.microsoft.com/en-us/microsoftservices/support.aspx) or another third party support provider.
```

Comment:
This is good, but if we rearrange things a bit it can be even stronger:

> Although this persistence will run on the free version of the above engines, i.e. [SQL Server Express](https://www.microsoft.com/en-au/sql-server/sql-server-editions-express) and [MySQL Community Edition](https://www.mysql.com/products/community/), it is strongly recommended to use commercial versions for any production system.